### ISSUE TITLE
A convenient way to get array of markers of a marker that are selected/highlighted

### ISSUE THREAD

**Issue by angieskazka** [vizabi/vizabi-reactive#22]
Maybe there is one, i just don't know

so select/filter encodings have a filter
that filter can be set by .markers or by .dimensions, right?
and if the filter is set by .dimensions then .markers is empty

so if i want to get an array of markers that are selected in a marker i would do something like

this.maker.dataArray.filter(d => this.marker.encoding.get("selected").filter.has(d))

can there be an easier way? such as

this.marker.encoding.get("selected").dataArray

**Comment by jheeffer** [comment]
Good point. Could with an argument where you give an array of markers which then gets filtered, with default of that array being the encoding's marker's data array.
